### PR TITLE
Update ABIDE_I bigtime

### PR DESCRIPTION
**Pull request by ericearl**
- Super-simplified data_convert.py
- Updated dictionary name and first dictionary field
- Updated ABIDE_I/README.md to describe SITE_ID source
- Updated dictionary names of ABIDE_II, HCP, PNC, and UKB

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -13,10 +13,11 @@ ABIDE_I/Phenotypic_V1_0b.csv
 ABIDE_I/fields_in*
 ABCD/data/
 HBN/Data Dictionaries/
+HCP/RESTRICTED_*
+HCP/unrestricted_*
+NKI/assessments_documentation/
```
Comment (ericearl) [inline .gitignore]: Not just one user AND alphabetized.

File: ABIDE_I/data_convert.py [hunk 2]
```diff
@@ -5,34 +5,18 @@
...
-from pathlib import Path
-
 import pandas as pd
+from pathlib import Path
 
+# file path handling
+HERE = Path(__file__).parent.resolve()
+INPUT = HERE.joinpath('Phenotypic_V1_0b.csv')
+OUTPUT = HERE.joinpath('phenotype', 'phenotype.tsv')
+status = OUTPUT.parent.mkdir(parents=True, exist_ok=True)
```
Comment (ericearl) [inline ABIDE_I/data_convert.py:16]: I think these are necessary for each script to simplify Path resolution.

**Comment by Arshitha** [comment]
@ericearl Since there'll be upcoming changes based on our meeting with @dmoracze , I'll add comments once it's done.

**Comment by ericearl** [comment]
Merging to get moving on integration. It will be tested again once integrated.

**Pull request merged**